core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-uve-toolbar/dot-uve-toolbar.component.spec.ts

```diff
@@ -4,7 +4,10 @@ import { MockComponent } from 'ng-mocks';
 import { of } from 'rxjs';
 
 import { HttpClientTestingModule, provideHttpClientTesting } from '@angular/common/http/testing';
-import { signal } from '@angular/core';
+import { DebugElement, signal } from '@angular/core';
+import { By } from '@angular/platform-browser';
```

Comment:
I believe in spectator we don't need `By` but I can be wrong

Reply:
We can use a byTestId() from spectator to get a ElementRef, but for this test we need to check the DebugElement
We have a lot of tests running like this, I'll see if we can use just spectator for the next tests/PR